react-function-components/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "react-function-components",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@babel/core": "^7.14.6",
+    "@babel/plugin-transform-react-jsx": "^7.14.5",
+    "babel-loader": "^8.2.2",
+    "webpack": "^5.43.0",
+    "webpack-cli": "^4.7.2"
+  },
+  "dependencies": {
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  },
+  "scripts": {
+    "build": "webpack"
+  }
```

Comment:
You can't create a second script object in the package.json, please move the `build` script into the above `scripts` object

Reply:
Please fix